### DISCUSSION TITLE
When will be the next release?

### DISCUSSION THREAD

**Discussion by andreas-menzl** [fossology/fossology#2845] [General]
Hi,

I noticed that the last release was in January, and since then, there have been numerous bug fixes and new features added to the master branch. I was wondering if there are any plans for an upcoming release, as we are particularly interested in some of the recent improvements.

It would be great to know if you have an approximate timeline for the next release. 
Thank you for all your efforts :)

**Comment by eschenbachertq** [comment]
I would also be interested to know when the final release 4.5.0 can be expected?
